### ISSUE TITLE
How to get predictions?

### ISSUE THREAD

**Issue by desireevl** [lonePatient/Bert-Multi-Label-Text-Classification#3]
Hi, if I wanted to get a list of what class the model predicts (for unseen data) to create a confusion matrix how would I go about this? I am not entirely sure how to interpret the output logits.

**Comment by desireevl** [comment]
Sorry, misread the output, I understand now, thanks for the great code!
